src/coreclr/jit/emitxarch.h

```diff
@@ -86,14 +89,19 @@ unsigned insSSval(unsigned scale);
 
 static bool IsSSEInstruction(instruction ins);
 static bool IsSSEOrAVXInstruction(instruction ins);
+static bool IsSSEOrAVXorAVX512Instruction(instruction ins);
```

Comment:
I wonder if this would be better as just `IsAvx512OrPriorInstruction`. Its a bit more readable and less verbose.

Reply:
Changed.